.devcontainer/docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+---
+# Dev container https://containers.dev/, e.g. for VS Code
+version: '2.4'
+services:
+  devcontainer:
+    extends:
+      file: "docker-compose.yml"
+      service: test
+    build:
+      dockerfile: .devcontainer/Dockerfile
+    links:
+      - db
+      - fakesentry
+      - redis-cache
+      - localstack
+      - oidcprovider
+      - statsd
+    entrypoint: ["sleep", "inf"]
```

Comment:
IIUC, normally docker containers live until their commands finish running and then shutdown. This is saying to keep the container running indefinitely.

Reply:
Something like this, though I'm not sure about the exact definition of the "lifetime" of a container. The container kind of keeps existing after the command finished (unless you use `--rm` or something similar on the command line), but it's not _running_ anymore. VS Code requires the container to be running, and this ensure it keeps running until it's shut down.